File: c/gemini/p116.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    char command[50];
    char *options = "-l";

    // 'sprintf' acts as the print function to format the string
    sprintf(command, "ls %s", options);

    // 'system' executes the command
    system(command);

    return 0;
}

File: java/gemini/p116.java
import java.io.IOException;

public class SystemCommand {
    public static void main(String[] args) {
        String options = "-l";

        // String.format creates the formatted string
        String command = String.format("ls %s", options);

        try {
            // Executing the command
            Process process = Runtime.getRuntime().exec(command);
            
            // Note: In a real Java app, you must read the process InputStream 
            // to see the output of 'ls'.
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}